roles/contiv_network/tasks/k8s_tasks.yml

```diff
@@ -5,20 +5,30 @@
   with_items:
     - defaults
 
+- name: ensure kubernetes dirs exist
+  file:
+    path: "{{ item }}"
+    recurse: yes
+    state: directory
+  with_items:
+    - "{{ k8s_manifests_dir }}"
+    - "{{ k8s_cert_dir }}"
+    - "{{ k8s_net_plugin_config_dir }}"
```

Comment:
Netplugin dir part is required. Why do we need the k8s_manifests & certs here? Netplugin wont work in an install which doesnt have these folders. And by having these created, we increase the chance that some one misconfigures the folder paths. As we depend on the content of these folders, and we dont create them, its better to fail early if they are not there.